driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTest.java

```diff
@@ -414,8 +420,36 @@ private void compareEvents(final UnifiedTestContext context, final BsonDocument
                 context.getEventMatcher().assertConnectionPoolEventsEquality(client, ignoreExtraEvents, expectedEvents,
                         listener.getEvents());
             } else if (eventType.equals("sdam")) {
-                TestServerMonitorListener listener = entities.getServerMonitorListener(client);
-                context.getEventMatcher().assertServerMonitorEventsEquality(client, ignoreExtraEvents, expectedEvents, listener.getEvents());
+
+                // Sdam tests also include topology events, so we need to separate them
```

Comment:
```suggestion
                // SDAM tests also include topology events, so we need to separate them
```
seems we used `Sdam` for code, but in comment `SDAM` might be more natural